cnd/src/swap_protocols/facade.rs

```diff
@@ -87,14 +166,12 @@ pub struct Facade {
     pub swarm: Swarm,
     // We currently only support Han-HALight, therefor 'alpha' is Ethereum and 'beta' is Lightning.
     pub alpha_ledger_states: Arc<LedgerStates>,
-    pub beta_ledger_states: Arc<halight::States>,
+    pub halight_states: Arc<halight::States>,
```

Comment:
Why was this renamed?

Reply:
Because the name is wrong, its an artifact of rfc003.